chinglee-iot/FreeRTOS
# Coverage for vTaskSuspend

Description
-----------
Test for vTaskSuspend excluding the config asserts.
Test Steps
-----------
cd FreeRTOS/FreeRTOS/Test/CMock$

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
